.github/workflows/ci.yaml

```
@@ -0,0 +1,57 @@
+name: CI
+on: push
+
+jobs:
+  frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v4
+        with:
+          version: 9.15.3
+          run_install: false
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: 'pnpm'
+
+      - name: 'Create env file'
+        run: |
+          touch .env
+          echo DATABASE_URL=postgres://backend:password@localhost:5432/hack4krak >> .env
+
+      - name: 'Install dependencies'
+        run: | 
+          pnpm install
+
+      - name: "Run tests and lint"
+        run: |
+          pnpm test
```

Also split into 2 sepate tasks.
```suggestion
          pnpm fronted:test
```